irods/test/data_obj_test.py

```diff
@@ -412,8 +412,9 @@ def test_obj_does_not_exist(self):
 
 
     def test_create_from_invalid_path__250(self):
-        possible_exceptions = { ex.CAT_UNKNOWN_COLLECTION:  (lambda serv_vsn : serv_vsn >= (4,2,9)),
-                                ex.SYS_INVALID_INPUT_PARAM: (lambda serv_vsn : serv_vsn <= (4,2,8))
+        possible_exceptions = { ex.SYS_INVALID_INPUT_PARAM: (lambda serv_vsn : serv_vsn <= (4,2,8)),
+                                ex.CAT_UNKNOWN_COLLECTION:  (lambda serv_vsn : (4,2,9) <= serv_vsn < (4,3,0)),
+                                ex.SYS_INVALID_FILE_PATH:   (lambda serv_vsn : (4,3,0) <= serv_vsn)
```

Comment:
> don't think [irods/irods@28e86867](https://github.com/irods/irods/commit/28e86867) could be it b/c the change occurred since 4.2.11

If this is true, shouldn't the comparisons be against 4.2.12 rather than 4.3.0? Like this...
```python
        possible_exceptions = { ex.SYS_INVALID_INPUT_PARAM: (lambda serv_vsn : serv_vsn <= (4,2,8)),
                                ex.CAT_UNKNOWN_COLLECTION:  (lambda serv_vsn : (4,2,9) <= serv_vsn < (4.2,12)),
                                ex.SYS_INVALID_FILE_PATH:   (lambda serv_vsn : (4,2.12) <= serv_vsn)
```

Reply:
will check.

Reply:
Is this inconsistent with what we thought was happening?